internal/pkg/service/buffer/model/schema/schema_test.go

```diff
@@ -0,0 +1,100 @@
+package schema
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type keyTestCase struct{ actual, expected string }
+
+func TestSchema(t *testing.T) {
+	t.Parallel()
+	cases := []keyTestCase{
+		{
+			Configs().Prefix(),
+			"config/",
+		},
+		{
+			Configs().Receivers().Prefix(),
+			"config/receiver/",
+		},
+		{
+			Configs().Receivers().InProject(123).Prefix(),
+			"config/receiver/123/",
+		},
+		{
+			Configs().Receivers().InProject(123).ID("my-receiver").Key(),
+			"config/receiver/123/my-receiver",
+		},
+		{
+			Configs().Exports().Prefix(),
+			"config/export/",
```

Comment:
V tomto commite definujem schemu etcd klucov.

Tu sa testuju iba string hodnoty, `Prefix/Key` (kazda chain ma bud `Key`, alebo `Prefix` metodu, nie obe).

Okrem toho, ze to vie vratit key/prefix ako string, tak kazda chain ma definovane operacie z predchadzajuceho commitu (a tam su aj testovane), napr. `Configs().Receivers().InProject(123).DeleteAll().Do(...)`